test/EndToEndTests/Framework/Core/Client/ClientExceptionUtil.cs

```diff
@@ -25,8 +25,7 @@ public static class ClientExceptionUtil
         /// <returns>The server error message.</returns>
         public static string ExtractServerErrorMessage(DataServiceQueryException exception)
         {
-            string contentType = exception.Response.Headers[HttpHeaders.ContentType];
-
+            string contentType = exception.Response.Headers[HttpHeaders.ContentType]?.Replace(" ", string.Empty);

```

Comment:
Am curious to understand why we are adding this code `.Replace(" ", string.Empty);`
I have seen it in multiple places.

Reply:
to remove any spaces.